library/general/src/lib/ui/installation_dialog.rb

```diff
@@ -135,12 +135,26 @@ def create_dialog
         dialog_title,
         dialog_content,
         help_text,
-        Yast::GetInstArgs.enable_back,
-        Yast::GetInstArgs.enable_next || Yast::Mode.normal
+        enable_back,
+        enable_next
       )
       true
     end
 
+    # Whether the back button is enabled
```

Comment:
please document behavior in this implementation ( so read options from client arguments ).